Sources/MapboxCoreNavigation/RerouteControllerDelegate.swift

```diff
@@ -17,17 +17,17 @@ protocol ReroutingControllerDelegate: AnyObject {
 public enum ReroutingError: Error {
     /// Could not correctly process the reroute.
     case routeError
-    /// Could not compose correct request for rerouting.
-    case wrongRequest
+    /// Cause of reroute error is unknown.
+    case unknown
```

Comment:
`unknown` seems like it should be the first case. (Not that it particularly matters for an Error enumeration that isn’t CaseIterable.)

Renaming this enumeration case would be a backwards-incompatible change. We should keep the `wrongRequest` case around but alias it to another value. mapbox/mapbox-navigation-android#5784 aliases it to `unknown`, but the implementation in MapboxNavigationNative seems to have replaced every occurrence of `wrongRequest` with `invalidInput`. We don’t have an `invalidInput` case, but I think `routeError` would be analogous perhaps? It’d be OK to alias it to `unknown` for now but change it to `routeError` later.

Reply:
Yes. We were just looking at the failing tests and trying to get some additional info some circleci. We weren't going to keep this change as is.

Reply:
> mapbox/mapbox-navigation-android#5784 aliases it to `unknown`, but the implementation in MapboxNavigationNative seems to have replaced every occurrence of `wrongRequest` with `invalidInput`.

Per @LukasPaczos:

> On the Nav SDK side we only handle the response errors, and only for custom routers, that's why the `RouterErrorType.UNKNOWN` is predominantly used.